### DISCUSSION TITLE
DataTable with Filters - sizing doesn't work

### DISCUSSION THREAD

**Discussion by bhaidar** [primefaces/community#411] [PrimeVue] [answered]
Hi,

When I have a DataTable with size set to `p-datatable-small` or `p-datatable-normal`, the same behavior and same sizing.

Is there a way to make a DataTable with size small even when having filters?

https://codesandbox.io/s/primevue-demo-forked-l6n9t9?file=/src/App.vue

Thanks
Bilal

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

It seems there is a typo here. It should be `p-datatable-sm`. Also there is no `p-datatable-normal` class for DataTable.

**Reply by bhaidar** [reply]
Oh my bad! The docs use the size.class not size.value! Thank you
